### ISSUE TITLE
Does not switch forms between SWL and others 

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/radio-logger#11]
The stuff from line 220 onward in /src/components/MyNav.js belongs in the gray shaded area of the SWL logging and Amateur radio logging sections.

It doesn't switch so it may need routes. When you click on the appopriate section you would like to use.

I was not sure where to put this code for the forms, or to create a new component for each one and then it should create / render each form depending on which menu item your looking at.

Also I originally built the forms in native HTML not react. They are in the "RadioLogger" repo as hamform.html and index.html (in the root tree).  They need converted to REACT form components... 

These are complex forms that needs the data stored into postgre database(s).

**Comment by gbowne1** [comment]
These are similar forms. They should render under Logging > in this version

**Comment by idormain** [comment]
The radio logger solution from GitHub seems to be completely blank ? Is this meant to be the case? Also can you email me the ip and port number to the db and any credentials I may need. Also what database is it?

Cheers

Ivan

**Comment by gbowne1** [comment]
Yeah I can do that.  I sent you a Discord direct message with the details.

The DB is called radioLogger

**Comment by gbowne1** [comment]
I use VSCode and also did not use dotnet new so it did not build .sln/.suo if its empty, there isn't one.  I sometimes stick in empty folders where they would go.

**Comment by gbowne1** [comment]
This is working now!

 -- Added 'exact path' to routes
 -- Added correct routing
 -- Added content to fieldsets

**Comment by gbowne1** [comment]
No longer works with current version of dependencies in package.json in a fresh install of this repository

App.js needs refactored to include not using Switch due to `react-router-dom`

**Comment by gbowne1** [comment]
This sort of works.. some routes need new things created.  

Nothing renders when the app is opened so it appears blank till you open a nav item from the drawer(s)..
